Overture.playground/Pages/Basics.xcplaygroundpage/Contents.swift

```diff
@@ -38,7 +38,31 @@ let capitalizeUserName = setUserName(capitalized(Locale(identifier: "en")))
 let setUserAge = prop(\User.age)
 let celebrateBirthday = setUserAge(incr)
 
-with(User(name: "blob", age: 1), concat(
+let blob = User(name: "Blob", age: 42)
+
+with(blob, concat(
   capitalizeUserName,
   celebrateBirthday
 ))
+
+let blobJr = with(blob, concat(
+  mver(\.name) { $0 += ", Jr." },
+  mver(\.age) { $0 -= 30 }
+))
+
+let blobSr = with(blob, concat(
+  mver(\.name) { $0 += ", Sr." },
+  mver(\.age) { $0 += 30 }
+))
+
+let users = [
+  blob,
+  blobJr,
+  blobSr
+]
+
+users.reduce(0, combining(get(\.age), +))
+users.reduce(into: 0, combining(get(\.age), +=))
+
+users.sorted(by: their(get(\.age)))
```

Comment:
This has me thinking on how these units combine. It'd be nice to compose comparators...

``` swift
users.sorted(by: their(get(\.age), get(\.name))
```